charts/m4d/templates/m4d-config.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: m4d-config
+data:
+  {{- with .Values.config }}
+  SECRET_PROVIDER_URL: "http://secret-provider.m4d-system:5555/get-secret"
+  SECRET_PROVIDER_ROLE: "demo"
+  VAULT_ADDRESS: {{ .vault.address }}
+  VAULT_DATASET_HOME: {{ .vault.datasetHome }}
+  VAULT_DATASET_MOUNT: {{ .vault.datasetMount }}
+  VAULT_USER_HOME: {{ .vault.userHome }}
+  VAULT_USER_MOUNT: {{ .vault.userMount }}
+  VAULT_TTL: {{ .vault.ttl }}
```

Comment:
There was also VAULT_AUTH equal to "kubernetes". Either remove the code that is using it, or return this var.

Reply:
The code that uses it relates to the secret-provider. I will remove it in a follow up patch after all the modules will call directly to vault to get the dataset creds.

Reply:
The configmap includes all, but variables that are soon to be removed don't have to be templated.

Reply:
OK sure. Hopefully it will be removed soon. If there will be a need I will add it.

Reply:
Ok on second though I added VAULT_AUTH.